bin/webpack.js

```diff
@@ -154,7 +154,13 @@ if (installedClis.length === 0) {
 			});
 	});
 } else if (installedClis.length === 1) {
-	require(installedClis[0].package); // eslint-disable-line
+	const path = require("path");
+	const readJson = require("read-package-json");
```

Comment:
this seems like needless bloat. node parses json files by default and returns an object.

Reply:
The reason for read-package-json is to normalize the shape of the `pkg.bin` object, since it could be either a string or an object. I'd be ok with rewriting this to do something like
```js
let binPath;
if (typeof pkg.bin === 'string') {
  binPath = pkg.bin;
} else {
  const binName = Object.keys(pkg.bin)[0];
  binPath = pkg.bin[binName];
}
require(path.resolve(path.dirname(pkgPath), binPath));
```

Reply:
In the case of webpack-cli, we want `bin['webpack-cli']`. I can go ahead and hard-code the name for each of the two CLI packages.